### PR TITLE
Added IsDelete() method for watch events

### PR DESCRIPTION
**Pull request by tinx**
It is now possible to check for the delete event type the
in the same fashion as testing for create and modify was
already done.

# Contributing guidelines

Please read our [contribution workflow][contributing] before submitting a pull request.

[contributing]: https://github.com/coreos/etcd/blob/master/CONTRIBUTING.md#contribution-flow

### CODE REVIEW

**Comment by hexfusion** [comment]
Hi @tinx, Travis will fail because you need to format commit message like $package: $message outside any other failures. Thanks!

File: clientv3/integration/watch_test.go [hunk 1]
```diff
@@ -620,11 +620,13 @@ func TestWatchEventType(t *testing.T) {
 		isModify: true,
 	}, {
 		et: clientv3.EventTypeDelete,
+		isDelete: true,
```
Comment (spzala) [inline clientv3/integration/watch_test.go]: Doesn't `isDelete` need to be a part of struct here, https://github.com/coreos/etcd/blob/5639bdfd16ae973142b4d7285a22628de1d6c37a/clientv3/integration/watch_test.go#L611 ?

**Comment by spzala** [comment]
@tinx hello, do you still plan to work on this PR? Thanks!

**Comment by xiang90** [comment]
Closing since no one is working on it.

**Pull request closed without merging**